### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by berrydenhartog**
# Description

Added Editorconfig to standardize editor settings

### CODE REVIEW

**Comment by uittenbroekrobbert** [comment]
> # Description
> 
> Please include a summary of the changes and the related issue.

... is this PR missing the above information or is this not needed (just wondering).. the branch name suggets it adds an editorconfig, but it also included a dev container for running a (local?) (test?) mkdocs project.. so, looks good to me, but wondering about the PR comment :)

**Comment by berrydenhartog** [comment]
Good catch @uittenbroekrobbert . Fixed the commits

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```
Comment (anneschuth) [inline .editorconfig:12]: I'd set this to true

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
```
Comment (anneschuth) [inline .editorconfig:11]: I'd set this to true

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+insert_final_newline = false
+
+[*.md]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = false
```
Comment (anneschuth) [inline .editorconfig:17]: I'd set this to true

**Review by anneschuth — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 2 above)
Comment (berrydenhartog) [reply .editorconfig]: Done

File: .editorconfig (hunk 1 above)
[the remark this answers is not in the document]
Comment (berrydenhartog) [reply .editorconfig]: Done

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+[*.y{a,}ml]
+indent_style = space
```
[the remark this answers is not in the document]
Comment (berrydenhartog) [reply .editorconfig]: Done

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
+
+[*.md]
+indent_style = space
+indent_size = 4
```
[the remark this answers is not in the document]
Comment (berrydenhartog) [reply .editorconfig]: done

File: .editorconfig [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
```
[the remark this answers is not in the document]
Comment (berrydenhartog) [reply .editorconfig:6]: Done

**Review by robbertbos — APPROVED** [review]

**Review by anneschuth — APPROVED** [review]

**Pull request merged**